spec/cb/responses/api_response_spec.rb

```diff
@@ -117,6 +119,15 @@ def extract_models
             expect(DumbValidResponse.new(valid_input_hash).respond_to?(:errors)).to eq true
           end
         end
+        
+        context 'missing required field' do
+          it 'throws ExpectedResponseFieldMissing with specific error message' do
+            expect{DumbValidResponse.new(other_input_hash)}.
+                to raise_error(Cb::ExpectedResponseFieldMissing) do |ex|
```

Comment:
We've upped our max line length to 120 characters, so this whole thing could go on the same line as the above:

```rb
expect { DumbValidResponse.new(other_input_hash) }.to raise_error(Cb::ExpectedResponseFieldMissing) do |ex|
```